src/legacy/core_plugins/kbn_vislib_vis_types/public/pie.js

```diff
@@ -80,7 +80,7 @@ export default function HistogramVisType(Private) {
         {
           group: 'buckets',
           name: 'segment',
-          icon: 'fa fa-scissors',
+          icon: 'cut',
```

Comment:
I would just remove these icons. They don't lend anything to the description of the action.